### ISSUE TITLE
Some functions don't seem to work from time to time.

### ISSUE THREAD

**Issue by DUSKvsDAWN** [finndev/PokeBuddy#1039]
Either it doesn't search for Pokemon, doesn't catch them, or I don't get XP and items from visiting Pokestops. :(

**Comment by 200serial** [comment]
softban

**Comment by DUSKvsDAWN** [comment]
Yeah, I thought so. Guess I shouldn't try to feel that overpowered and turn on teleport and such.

**Comment by AlexCtasy** [comment]
@DUSKvsDAWN I was with same issue yesterday, and I was unbanned. Auto unbanned.. I waited like 12hours or more.
